Packs/SailPointIdentityNow/Integrations/SailPointIdentityNow/SailPointIdentityNow.py

```diff
@@ -0,0 +1,622 @@
+
+    :type requested_for: ``str``
+    :param requested_for: The identity that the activity was requested for (me indicates current user).
+
+    :type requested_by: ``str``
+    :param requested_by: The identity that requested the activity (me indicates current user).
+
+    :type regarding_identity: ``str``
+    :param regarding_identity: The specified identity will be either requester or target of account activity (me
+    indicates current user).
+
+    :type type: ``str``
+    :param type: Type of account activity.
+
+    :type offset: ``int``
+    :param offset: Offset into the full result set. Usually specified with limit to paginate through the results.
+
+    :type limit: ``int``
+    :param limit: Max number of results to return. Maximum of 250.
+
+    :return: Account object (JSON) corresponding to the filter parameters. Empty JSON if the event was not found in
+    IdentityNow.
+    """
+    if not offset or offset < 0:
+        offset = 0
+
+    if not limit or limit > MAX_LIMIT:
+        limit = MAX_LIMIT
+
+    params = {'offset': offset, 'limit': limit}
+    if id is not None:
+        url = ''.join((IDN_BETA_ACCOUNT_ACTIVITIES_EXT, '/', id))
+    else:
+        url = IDN_BETA_ACCOUNT_ACTIVITIES_EXT
+        if requested_for is not None:
+            params.update({'requested-for': requested_for})
+        if requested_by is not None:
+            params.update({'requested-by': requested_by})
+        if regarding_identity is not None:
+            params.update({'regarding-identity': regarding_identity})
+        if type is not None:
+            params.update({'filters': ''.join(('type eq "', type, '"'))})
+    return client.send_request(url, "GET", params, None)
+
+
+def access_request_bulk(client: Client, request_type: str, requested_for=None, requested_items=None):
+    """
+    Grant or revoke access request for a list of objects(access profiles or roles) for a list of users.
+    Command(s): identitynow-request-grant, identitynow-request-revoke
+
+    :type client: ``Client``
+    :param client: SailPoint client
+
+    :type request_type: ``str``
+    :param request_type: Type of access request(GRANT_ACCESS or REVOKE_ACCESS).
+
+    :type requested_for: ``list``
+    :param requested_for: List of Identity Id's.
+
+    :type requested_items: ``list``
+    :param requested_items: List of objects(access profiles or roles).
+
+    :return: Success or failure status.
+    """
+    if request_type is None or requested_for is None or requested_items is None:
+        return None
+
+    json_data = {
+        "requestedFor": requested_for,
+        "requestType": request_type,
+        "requestedItems": requested_items,
+        "clientMetadata": {}
+    }
+    response = client.send_request(IDN_BETA_ACCESS_REQUEST_EXT, "POST", None, json_data)
+    if response is not None and 200 <= response.status_code < 300:
+        return 'Access request was successful!'
+    elif 'detailCode' in response.json():
```

Comment:
Can you please explain this?  why are you returning a json and not a commandResult?

Reply:
For this API, the response above is the same as what is returned by the API.